src/main/java/org/jenkinsci/plugins/docker/traceability/core/DockerTraceabilityRootAction.java

```diff
@@ -568,18 +571,69 @@ private synchronized void load() {
             LOGGER.log(Level.SEVERE, "Failed to load the configuration from config path = "+config,e);
         }
     }
-    
+
+    /**
+     * Represents the different response formats (JSON, pretty JSON, XML).
+     */
+    private enum ResponseFormat {
+        JSON("json", "application/json;charset=UTF-8"),
+        PRETTYJSON("json-pretty", "application/json;charset=UTF-8"),
+        XML("xml", "application/xml;charset=UTF-8");
```

Comment:
I'll change it but this RFC defines both as valid for "document entities" https://www.ietf.org/rfc/rfc3023.txt